index.html

```diff
@@ -4375,27 +4726,29 @@ <h3 id="Warrior">Warrior</h3>
                      making hostiles aggressive. </strong>
                   <p>
                      Until the start of your next turn, you become <a class="condition" data-tooltip-container>resistant</a> to all physical
-                     damage. All enemies within 10ft of you become <a class="condition" data-tooltip-container>taunted</a>. 1 WILL point to
+                     damage and one other damage type of your choice. All enemies within 15ft of you become 
+                     <a class="condition" data-tooltip-container>taunted</a>. 1 WILL point to
                      resist the <a class="condition" data-tooltip-container>taunt</a>.
                </td>
-               <td>E, 10ft, All within range (AoE)
+               <td>E, 15ft, All within range (AoE)
                </td>
                <td>1
                </td>
             </tr>
             <tr>
                <td>1
                </td>
-               <td>Ignore Pain
+               <td>Aggressive Proximity
                </td>
-               <td><strong>There is no need for evasion or armour when you can shrug off
-                     pain without issue. </strong>
+               <td><strong>Take advantage of enemies that approach you.</strong>
                   <p>
-                     You may use your STR bonus to calculate your AC as if it were DEX.
+                     Spend your reaction in addition to the AP cost. Then, when a creature moves into a tile within 5ft of
```

Comment:
This should be a react ability that unlocks it for the rest of the turn imo.  A bit cleaner and wasting the ability use if it doesn't happen is a feels bad

Reply:
Yeah I did consider this, the reason I didn't is because it's 1AP damage with high potential output. Making it more flexible could make it just a bit too good, esp at level 1. It's gonna be one that people want to coordinate with to get enemies next to you, but it should be hard to get the payoff